main.go

```diff
@@ -252,6 +254,9 @@ func main() {
 	kingpin.HelpFlag.Short('h')
 	kingpin.Parse()
 
+	log.Base().SetLevel(*logLevel)
```

Comment:
Hmm, fair point, but that would be linking common/log to kingpin. I don't think we should add that dependency.